frontend/apps/filemanager/filemanagerhistory.lua

```diff
@@ -51,8 +50,6 @@ function FileManagerHistory:fetchStatuses(count)
     for _, v in ipairs(require("readhistory").hist) do
         if v.dim then
             status = "deleted"
-        elseif self:isFileCurrentlyOpened(v.file) then
-            status = "reading"
         else
             if DocSettings:hasSidecarFile(v.file) then
                 local docinfo = DocSettings:open(v.file) -- no io handles created, do not close
```

Comment:
If in the current opened book session, I changed the book status (ie. On Hold), will this fetch the right status from the docsettings ? That is, is the docsettings flushed when it is updated ? Or would you need to fetch it from self.ui.docsettings if isFileCurrentlyOpened() ?

Reply:
Yes.
Status is flushed on closing the BookStatusWidget (it is even flushed on changing the status, but I want to change this behaviour and flush on closing only).
Status is fetched on opening the History.